content/tutorials/drift-detection/index.md

```diff
@@ -0,0 +1,198 @@
+```
+
+Now you can view your new stack in Pulumi Cloud by following the **View in Browser** link. Click on the **Drift** tab and you will see that Drift is unavailable as your stack's deployments are not yet configured. Click on the **Configure deployments setting** to see your Deployments source control settings.
+
+To use Drift Detection with Deployments you will first need to create a new GitHub repository.
+
+## Create a new repository on GitHub
+
+1. Go to [GitHub](https://github.com/new) and create a new repository manually.
+2. Copy the repository URL provided by GitHub.
+3. Replace the origin with your new GitHub repo using the following commands:
+
+```bash
+git init
+git add .
+git commit -m "Initial commit for my Pulumi Drift tutorial"
+git branch -M main
+git remote add origin https://github.com/your-username/learn-pulumi-drift-detection.git
+git push -u origin main
+```
+
+## Configure Drift Detection
+
+Now you will return to your browser to set your deployment configuration source control settings. Navigate to your stack `Settings` and `Deploy` tab. Next, configure the required Deployment settings, including your source control.
+
+1. Add your GitHub organization/repository and branch you created.
+
+2. Add your Secret environment variables, including `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end with Pulumi and can be set on each stack. However, by creating an [environment](/docs/pulumi-cloud/esc) with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+Click **Save deployment configuration**.
+
+![Animation of how to configure drift detection in Pulumi](./pulumi-configure-drift.mp4)
+
+Once you have configured Pulumi Deployments, open the Actions menu and choose `Detect Drift`, then click `Deploy` to validate your stack, program and deployments are all running before we introduce drift.
+
+![Screenshot of the Pulumi Cloud console to run a Drift Detection](./pulumi-detect-drift-action.png)
+
+Once the drift run is complete, you should see a new Drift check on the stack `Drift` Tab
+
+## Introduce drift
+
+To see Drift Detection in action, you will now introduce drift by manually making changes to your newly created EC2 resource, in this case by adding a new tag.
+
+Log in to the AWS Management console with your browser and ensure you are in the correct AWS region where your Pulumi stack is deployed.
+
+Modify your EC2 Instance by going to the EC2 Dashboard, then select the EC2 instance named `webserver` and click on the **Tags** tab. Next, add a tag by clicking the **Manage tags** button. Then click the **Add new tag** button with the key `Description` and value of `Pulumi Drift Detection`. Finally, **Save** the changes.
+
+## Run a Drift Detection deployment
```

Comment:
Is that really the correct way to say it? Should it isntead be "Run Drift Detection" or "Detect drift" or something that is more about the goal of the action rather than the feature of the product?

Reply:
agreed here.

Reply:
agreed here

Reply:
thx; fixed up